src/components/HeaderProduct/HeaderProduct.stories.tsx

```diff
@@ -150,6 +150,13 @@ const partyList: Array<PartyEntity> = [
     name: "Ente senza stemma",
     productRole: "Referente amministrativo",
   },
+  {
+    id: "14",
+    logoUrl: `${cdnPath}172960361.png`,
+    name: "Amministrazione Comunale",
+    productRole: "Referente amministrativo",
+    topicName: "Comune di Castelfranco Emilia",
```

Comment:
Proporrei di cambiare la key da `topicName` a `parentName` (e quindi `parentPartyName`) in modo da chiarire che, se presente, rappresenta l'ente apicale del quale è parte l'ente corrente

Reply:
ok, la PR è stata aggiornata con il suggerimento richiesto.

Reply:
Grazie mille! Domanda, mi è venuto in mente dopo: sull'API V2 Self Care ora il parametro si chiama sempre `topicName`, giusto?

Reply:
si esatto :) scusa il ritardo nella risposta